src/mavis/convert/vcf.py

```diff
@@ -122,6 +123,46 @@ def parse_bnd_alt(alt: str) -> Tuple[str, int, str, str, str, str]:
         raise NotImplementedError('alt specification in unexpected format', alt)
 
 
+def handle_edge_case(std_row: Dict) -> Dict:
+    """
+    Checks for known errors caused by callers in IMPRECISE calls, that leveraged uncertainty from the CIPOS/CIEND/CILEN fields.
+
+    bp1_s = breakpoint1 start
+    bp1_e = breakpoint1 end
+    bp2_s = breakpoint2 start
+    bp2_e = breakpoint2 end
+
+    Insertion edge cases include:
+    Scenario 1: bp1_s = 1890, bp1_e = 2000, bp2_s = 1900, bp2_e = 1900. bp1_e > bp2_s
+    Scenario 2: bp1_s = 1890, bp1_e = 2000, bp2_s = 1800, bp2_e = 2000. bp1_e > bp1_s
+    Scenario 3: bp1_s = 1950, bp1_e = 2000, bp2_s = 1900, bp2_e = 3000. bp1_s > bp2_s
+    """
+
+    try:
+        if (
+            std_row['break1_position_end'] > std_row['break2_position_end']
```

Comment:
could you make the comments here more descriptive? ex. what does scenario 1 mean?